build/dockerfiles/rhel.Dockerfile

```diff
@@ -44,21 +44,14 @@ COPY ./build/dockerfiles/content_set*.repo /etc/yum.repos.d/
 COPY ./build/dockerfiles/rhel.install.sh /tmp
 RUN /tmp/rhel.install.sh && rm -f /tmp/rhel.install.sh
 
-COPY ./build/scripts ./arbitrary-users-patch/base_images /build/
+COPY ./build/scripts /build/
 COPY ./devfiles /build/devfiles
 WORKDIR /build/
 
-# Registry, organization, and tag to use for base images in dockerfiles. Devfiles
-# will be rewritten during build to use these values for base images.
-ARG PATCHED_IMAGES_REG="quay.io"
-ARG PATCHED_IMAGES_ORG="eclipse"
-ARG PATCHED_IMAGES_TAG="next"
-RUN TAG=${PATCHED_IMAGES_TAG} \
-    ORGANIZATION=${PATCHED_IMAGES_ORG} \
-    REGISTRY=${PATCHED_IMAGES_REG} \
-    ./update_devfile_patched_image_tags.sh
 RUN ./check_mandatory_fields.sh devfiles
+
 RUN if [[ ${USE_DIGESTS} == "true" ]]; then ./write_image_digests.sh devfiles; fi
```

Comment:
see above -- remove this. No need for digest pinning in devfile reg -- it's computed from operator CSV env vars are runtime.